packages/material-ui/src/ButtonBase/ButtonBase.js

```diff
@@ -151,7 +151,7 @@ const ButtonBase = React.forwardRef(function ButtonBase(props, ref) {
     'stop',
     (event) => {
       if (focusVisible) {
-        onBlurVisible(event);
+        onBlurVisible();
```

Comment:
This is currently not needed but will be in the future with React 17. `onBlurVisible` already has a notice so this shouldn't be changed.